docs/installation.md

```diff
@@ -3,48 +3,60 @@
 > This instruction is mainly based on Linux:Ubuntu
 ## 0. Prerequisites 
 Install Go 
-> You can skip this part if you already have installed go version > 1.18 on your system. 
+> You can skip this part if you already have installed go version > 1.18+ on your system. 
 
-Please, check latest version or check on VATZ module and follow these [instructions](https://go.dev/doc/install) to install. 
+Please, check [latest version](https://github.com/dsrvlabs/vatz/releases) or check on [VATZ module](https://github.com/dsrvlabs/vatz/blob/main/go.mod) and follow these [instructions](https://go.dev/doc/install) to install. 
 
 
 ## 1. VATZ 
-1. Clone VATZ repository
-    ```
-    $ git clone git@github.com:dsrvlabs/vatz.git
-    ```
-2. Compile VATZ
-    ```
-    $ cd vatz
-    $ make
-    ```
-    You will see binary named `vatz`
+### 1. Clone VATZ repository
+
+  ```
+  $ git clone git@github.com:dsrvlabs/vatz.git
+  ```
+
+### 2. Compile VATZ
+
+  ```
+  $ cd vatz
+  $ make
+  ```
+You will see binary named `vatz`
 
   
-3. Initialize VATZ
-    ```
-    $ ./vatz init
-    ```
-    You will see config file named default.yaml once you initialize VATZ.
+### 3. Initialize VATZ
+```
+$ ./vatz init
+```
+You will see config file named default.yaml once you initialize VATZ.
+
+```
+$ ./vatz init --all
+```
+You can also use `--all` flag to add all default setting of [official plugins](https://github.com/dsrvlabs/vatz/blob/main/docs/installation.md#2-vatz-plugin) to config file. For more details, please check [cli.md](https://github.com/dsrvlabs/vatz/blob/main/docs/cli.md) file or use `--help` flag. 
 
 ## 2. VATZ Plugin
 
-1. Copy the plugin address
-  > **Plugin Repositories**
-  > - [vatz-plugin-sysutill](https://github.com/dsrvlabs/vatz-plugin-sysutil) 
-  >   - cpu_monitor
-  >   - disk_monitor
-  >   - mem_monitor
-  > - [vztz-plugin-cosmoshub](https://github.com/dsrvlabs/vatz-plugin-cosmoshub)
-  >   - node_active_status
-  >   - node_block_sync
-  >   - node_governance_alarm
-  >   - node-is_alived
-  >   - node_peer_count
+### 1. Copy the plugin address
 
-  We officially provide 3 plugins for System Utility and 5 plugins for CosmosHub node. But you can easily develop custom plugins for the feature that you want with [VATZ SDK](https://github.com/dsrvlabs/vatz/tree/main/sdk).
+  We officially provide 3 plugins for System Utility and 5 plugins for CosmosHub node. 
```

Comment:
We have 4 plugins for System Utility now.

Reply:
fixed